eogrow/pipelines/merge_samples.py

```diff
@@ -43,7 +44,6 @@ class Schema(Pipeline.Schema):
         )
         suffix: str = Field("", description="String to append to array filenames")
         workers: int = Field(1, description="Number of threads used to load data from EOPatches in parallel.")
```

Comment:
should we rename this parameter to be more in line with the number of threads for loading?  `threads`/`loading_threads`/`threads_per_worker`?